python/ray/air/examples/dreambooth/dataset.py

```diff
@@ -41,65 +53,97 @@ def _tokenize(prompt):
     class_prompt_ids = _tokenize(args.class_prompt)[0]
     instance_prompt_ids = _tokenize(args.instance_prompt)[0]
 
-    # Image preprocessing.
-    # Instance and class images used by this example are in sizes 700x700
-    # and 512x512 respectively.
-    # Depending on the sizes of actual training images, there may need to be a
-    # transforms.Resize() step as well.
+    # START: image preprocessing
     transform = transforms.Compose(
         [
             transforms.ToTensor(),
+            transforms.Resize(
```

Comment:
Interesting. Does this upsampling method affect the fine-tuning performance?

Reply:
I copied this one from HF. Haven't experimented too much with this